pkgs/flox-src/default.nix

```diff
@@ -0,0 +1,17 @@
+{}:
+builtins.path {
```

Comment:
non-blocking: The new lib.filesets API is also possible to use here?

Reply:
sure, i just copied this out, but filesets are less confusing than this.
rather do that separately